File: problem_forty_seven.js
//find lost number
var inp= 10
var lst =[1,2,3,5,6,7,9,10]
function MyFunction(inp,lst){
    for (var i=1; i<inp+1;i++){
        var boll = true
        for(var j = 0 ;j<lst.length;j++){
            if(lst[j]==i){
                boll = false
            }
        }if(boll){
            console.log(i)
        }      
    }
}
MyFunction(inp,lst)

File: problem_forty_seven.py
# find lost number
inp =10
lst=[2,3,5,1,6,8,9,10]
def MyFunction(inp,lst):
    for i in range(1,inp+1,1):
        bollen=False
        for j in range(len(lst)):
            if lst[j]==i:
                bollen= True
                break
        if bollen==False:
           print(i)

MyFunction(inp,lst)